frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTableCell.tsx

```diff
@@ -59,10 +62,30 @@ export function Cell({
       }}
       onClick={onClick}
     >
-      <div className={cx("px1 flex align-center", { "justify-end": isBody })}>
-        <Ellipsified>{value}</Ellipsified>
-        {icon && <div className="pl1">{icon}</div>}
-      </div>
+      <>
+        <div className={cx("px1 flex align-center", { "justify-end": isBody })}>
+          <Ellipsified>{value}</Ellipsified>
+          {icon && <div className="pl1">{icon}</div>}
+        </div>
+        {!!onResize && (
+          <Draggable
```

Comment:
Maayyybe giving the user a bit more affordance on making the column draggable might be nice.

Given them, say, 10 or 20 more pixels to the left and right so that the column becomes draggable.

A 2-min look at react-draggable in their Github did not teach me how to do that and try it myself.

Reply:
I think that's an easy css change on the handle element width, but here I was just copying the UX (and the basic implementation) from the `TableInteractive` - I think they should work the same - so that user knowledge transfers back and forth as much as possible.

Reply:
Gotcha. Thank you.